From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sherif Shalaby <sherif2222222@gmail.com>
Date: Tue, 13 Oct 2026 13:05:38 +0300
Subject: [PATCH] dng_sdk: drop sdk_version from libdng_sdk

DNG SDK 1.7.1 needs an sdk variant of libjpeg that libjpeg-turbo does not
provide on this branch; soong bootstrap fails.

Change-Id: Ie74175fc90dadd27b99f48f6df593ab7a68d82a3
---
 Android.bp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Android.bp b/Android.bp
index 6c8064b..1c57e7a 100644
--- a/Android.bp
+++ b/Android.bp
@@ -165,7 +165,6 @@ cc_defaults {
 cc_library {
     name: "libdng_sdk",
     host_supported: true,
-    sdk_version: "current",
     defaults: ["libdng_sdk-defaults"],
     vendor_available: true,
 
